wiki/Beatmap_ranking_procedure/en.md

```diff
@@ -14,11 +24,15 @@ Newer beatmappers usually need extensive modding to create beatmaps that meet th
 
 Before being nominated for ranking, a beatmap needs to receive [hype](/wiki/Beatmap/Hype) posts from at least five different users.
 
-## Nominations
+## Nominations {#nominations}
```

Comment:
what's this for?

Reply:
I want the redirects to always keep working, and for that it needs a permanent ID. if each one of the translations has a different section ID, it will throw a user to the start of the article instead.